### PR TITLE
Update psk tutorial to how it actually behaves

### PR DESCRIPTION
**Pull request by katejeffreys**

### CODE REVIEW

File: app/2.0/start/toolbox/set-up.md [hunk 1]
```diff
@@ -35,11 +39,11 @@ Polymer CLI is an all-in-one command line tool for Polymer projects. In this tut
     If you're missing any of these dependencies, follow the instructions in
     the following section from the Polymer CLI guide to learn how to install
     each one: [Install section from Polymer
-    CLI guide](/{{{polymer_version_dir}}}/docs/tools/polymer-cli#install).
+    CLI guide](/1.0/docs/tools/polymer-cli#install).
```
Comment (arthurevans) [inline app/2.0/start/toolbox/set-up.md]: Switch the 1.0 back to `{{{polymer_version_dir}}}`

File: app/2.0/start/toolbox/set-up.md [hunk 2]
```diff
@@ -98,4 +104,5 @@ a page to your app.
     href="create-a-page">Next step: Create a page</a>
 
 [toolbox]: /2.0/toolbox/
+[shared styles]: /1.0/docs/devguide/styling.html#style-modules
```
Comment (arthurevans) [inline app/2.0/start/toolbox/set-up.md]: This appears to be unused in this file, so we can cut it.

Comment (katejeffreys) [inline app/2.0/start/toolbox/set-up.md]: I can't tell what this refers to?

Comment (katejeffreys) [inline app/2.0/start/toolbox/set-up.md]: I think it's referring to this line

`[shared styles]: /1.0/docs/devguide/styling.html#style-modules`

Removed it.

File: app/2.0/start/toolbox/set-up.md [hunk 3]
```diff
@@ -35,11 +39,11 @@ Polymer CLI is an all-in-one command line tool for Polymer projects. In this tut
     If you're missing any of these dependencies, follow the instructions in
     the following section from the Polymer CLI guide to learn how to install
     each one: [Install section from Polymer
-    CLI guide](/{{{polymer_version_dir}}}/docs/tools/polymer-cli#install).
+    CLI guide]({{{polymer_version_dir}}}/docs/tools/polymer-cli#install).
```
Comment (arthurevans) [inline app/2.0/start/toolbox/set-up.md]: Oop. We need the leading / here or it's going to be a relative link, which won't work.

Comment (arthurevans) [inline app/2.0/start/toolbox/set-up.md]: Meaning the line `CLI guide]({{{polymer_version_dir}}}/docs/tools/polymer-cli#install)` needs to be:

`CLI guide](/{{{polymer_version_dir}}}/docs/tools/polymer-cli#install)`

Note leading slash.

**Pull request merged**